src/qumin/representations/segments.py

```diff
@@ -153,7 +168,7 @@ def regex(cls, sound, end=False):
         """
         if end:
             return cls._regexes_end[sound]
-        return cls._regexes[sound]
+        return "(?:" + cls._regexes[sound] + ")"
```

Comment:
I believe this is not necessary anymore, the cleaner fix has now been changed in master thanks to: e69ccb71e685

It is also in the dev branch: 

https://github.com/XachaB/Qumin/blob/bbfbbe39e26b6f45c869a0de81357882bcf62980/src/qumin/representations/segments.py#L130-L131